### DISCUSSION TITLE
大一體育這門課程

### DISCUSSION THREAD

**Discussion by 113607514** [1101-GS4719-C/Python-Community-GS4719-C#3687] [General]
大一的體育課，對我來說，最初是一種新鮮感，也帶著一絲不安。剛開始的時候，我總是擔心自己跟不上大家的步伐，畢竟長時間沒有系統地運動，身體狀況有些退步。不過，隨著課程進行，我逐漸發現體育課不僅是單純的體能鍛鍊，更多的是一種釋放壓力的方式。在忙碌的學業和生活之間，這成為我最放鬆的時光。無論是打籃球的汗水，還是跑步時的呼吸節奏，都讓我感覺自己重新找回了平衡。每一次突破自我、挑戰極限，不僅僅是為了身體的健康，更是對自信心的重建。體育課讓我明白，無論多忙，給自己一些運動的時間，是對身心最好的投資。而這段時間裡的每一次進步，也成為了我大學生活中最珍貴的小確幸。

**Comment by Mminniehuang** [comment]
我覺得大一體育的課程對於不會游泳的人真的是超級痛苦的，從上學期游不過21公尺被提醒說下學期游泳分數占40%，如果沒有游到50公尺可能會被當掉，到下學期瘋狂地被留下來以及瘋狂的補考和嗆到，就身心上都很煎熬。不過我覺得比起真的變得很會游泳很會運動，老師們看重的都是學習態度，就像很多人最後還是沒有游到終點，但堅持到最後一周還是願意練習和考試的同學還是及格了。
撐過大一體育之後的興趣選修就有趣多了，而且也蠻多元的，就看搶不搶的到心儀的課了！

**Comment by XiangLuen** [comment]
大一體育我認為是最麻煩的體育課，因為對於太久沒游泳或不會游泳的很痛苦，而且體育課通常在早八，然後還是冬天，睡不好就算了那天氣根本是會讓人生病，而且沒體力的人就算會游泳了25公尺也會沒力。另外我們班當時體育課上完還會接微積分，就讓你要趕快下課把頭髮用乾再去趕微積分課程，很難受

**Comment by lichilun** [comment]
我在大一體育課的時候，超級害怕下水，而且考試游泳距離也很長，幸好我們老師很有耐心一步一步教我們，讓許多人都可以度過這個大魔王，而且我們體育課是在禮拜五的最後一節，吹頭髮也沒有壓力，但我還是喜歡到大二體育多元選修，讓我可以嘗試喜歡的運動。

**Comment by SendaiHatsuki** [comment]
真的不太喜歡學校規定一定要上體育課，都到大學了應該要有更自由的選課，至於運動來說我覺得應該是自動自發的，學校可以開體育課給學生選，但不應該限制學生一定要修這堂課。

**Comment by a414414** [comment]
我覺得大一體育是一門非必要的課，而且課程還需要學游泳，我就完全沒有學過游泳，因此大一體育課對我來說滿痛苦的。
我覺得學校可以修改課程規劃，讓學生能自由選擇自己想要的體育課，而不是讓學生強制一定要上某門課程，已經到了大學選課應該是彈性的才對，學校應該在大一就開放學生自由選修體育課，這樣也能讓我們接觸更多運動，發展興趣。

**Comment by yue0903ZZ** [comment]
我覺得大一體育太雜了，對我來說就是涼課而已，但很浪費時間。
像其他學校從大一開始就能以自己的興趣開始選體育課了，真的覺得中央要改進一下。
然後游泳的時候，天氣都好冷，我從南部來的好討厭這樣的溫度還要下水。

**Comment by lilipop6405** [comment]
我認為大一體育不如換成跟高年級之後的體育課一樣，都可以自己選自己想要修的課。不然大一體育有的時候真的很無聊，雖然我不怕游泳，但是每次都要換衣服、洗澡真的很麻煩。還有的時候有些球類運動我也真的不感興趣，像是丟壘球之類的，我更喜歡去做其他的運動。

**Comment by jaychang1027** [comment]
大一體育我認為是最難受的一門課，對於不會游泳的我來說很痛苦，基本上是從0開始然後要在不到一個學期的時間內能夠游21公尺，而且體育課在早八，還是冬天，要早起就算了那天氣根本是會讓人生病，而且當時體育課上完還會接微積分，常常課上到一半就累到受不了進入夢鄉了。

**Comment by edawdewv** [comment]
大一體育對我來說超痛苦的。游泳這種東西對不會的人來說真的不是只靠「努力」就能馬上看到成果的，更別說還要面對壓力、嗆水、一次次補考，身體累、心理也會被磨到邊緣。但幸好我們這班老師其實看重的是學生願不願意撐到最後、那個態度。比起游得多遠，很多時候是看你有沒有堅持下來。幸好大二開始就能開始選自己喜歡的課，整個就變輕鬆很多了！

**Comment by Welly940504** [comment]
我覺得你這段分享真的很真誠，也很有共鳴感。很多人都會把體育課當成一門必修課來上，但你卻從中找到了放鬆和成長的空間，這點真的很棒。

特別是你提到「打籃球的汗水」和「跑步時的呼吸節奏」，那種專注在當下、什麼都不想的狀態，我也很能理解，真的很療癒。更讓人印象深刻的是你說「這不只是為了身體健康，更是對自信心的重建」，這句話真的打到我心裡了。有時候我們在學業或生活中遇到挫折，但透過一點一點的身體進步，會讓人重新相信自己做得到。

我也覺得，能在大學這段時間找到這樣的「小確幸」，是一件很值得珍惜的事。希望你之後也能一直保持這樣的態度，繼續在運動裡找到快樂和平衡！

**Comment by shrshu38** [comment]
我覺得大一體育課游泳佔的時間太多了，雖然學會游泳很重要，但有點壓縮到其他運動的機會。如果能把一半時間拿來嘗試像羽球、籃球、或排球，會讓整體體驗更豐富也更有趣。
